### ISSUE TITLE
[Bug] Tracking Protection pop-up's close button is unlabeled

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/fenix#8287]
## Steps to reproduce
Visit a site with tracking protection triggered.

### Expected behavior
Pop-up with close x button should have a labeled close button

### Actual behavior
Unlabeled button

### Device information

* Android device: Galaxy S10E
* Fenix version: Friday 2/7

**Comment by MarcoZehe** [comment]
This is still an issue in current Fenix Nightly, and also affects other onboarding Close buttons.

**Comment by asadotzler** [comment]
This seems to be fixed in the latest nightly build. Talkback reads "close button"
